README.md

```diff
@@ -10,7 +10,7 @@
 
 Gardener implements the automated management and operation of [Kubernetes](https://kubernetes.io/) clusters as a service and provides a fully validated extensibility framework that can be adjusted to any programmatic cloud or infrastructure provider.
 
-Gardener is 100% Kubernets-native and exposes its own Cluster API to create homogeneous clusters on all supported infrastructures. This API differs from [SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle)'s [Cluster API](https://github.com/kubernetes-sigs/cluster-api#cluster-api) that only harmonizes how to get to clusters, while [Gardener's Cluster API](https://gardener.cloud/docs/references/core/#core.gardener.cloud/v1beta1.Shoot) goes one step further and also harmonizes the make-up of the clusters themselves. That means, Gardener gives you homogeneous clusters with exactly the same bill of material, configuration and behavior on all supported infrastructures, which you can see further down below in the section on our K8s Conformance Test Coverage.
+Gardener is 100% Kubernetes-native and exposes its own Cluster API to create homogeneous clusters on all supported infrastructures. This API differs from [SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle)'s [Cluster API](https://github.com/kubernetes-sigs/cluster-api#cluster-api) that only harmonizes how to get to clusters, while [Gardener's Cluster API](https://gardener.cloud/docs/gardener/concepts/cluster-api/) goes one step further and also harmonizes the make-up of the clusters themselves. That means, Gardener gives you homogeneous clusters with exactly the same bill of material, configuration and behavior on all supported infrastructures, which you can see further down below in the section on our K8s Conformance Test Coverage.
```

Comment:
How does 

> Gardener's Cluster API](https://gardener.cloud/docs/gardener/concepts/cluster-api/)

make sense? Actually, the old link was the correct one (API documentation for the `Shoot` API.

Reply:
The old link was broken and I couldn't locate that part of the documentation anywhere on the site or in the repository files. Short of removing the link altogether, that was the closest link I could find. 

I will try to locate a more fitting link. If you know or can find a suitable replacement, feel free to post it here.

Reply:
@Kristian-ZH please help out here

Reply:
For now the link is missing from the website, I will fix it once the page gets added. I will put the PR in draft while this is being worked on.

Reply:
Yes, this depends on https://github.com/gardener/gardener/pull/4959. The page will be added when the PR is merged.

I will suggest to use github links instead of links to the website. If you read documentation from the website, docforge will take care of modifying the links in a way you will stay on the website (if the pointed page exists).
Pros:
- Currently if we restructure the documentation the link will no longer work on both sides (website and github), but if it points to github, it will continue to work.
- In my opinion it is better when u read documentation from once source to stay on this source, but not to jump from one source to another.

Reply:
I replaced the broken link with the correct relative one.